defold-rive/editor/src/rive.clj

```diff
@@ -806,7 +807,7 @@
 
                                        (merge {:node-id _node-id
                                                :renderable {:passes [pass/selection]}
-                                               :aabb (:aabb rive-main-scene)}
+                                               :aabb (if rive-main-scene (:aabb rive-main-scene) geom/null-aabb)}
```

Comment:
For the .rivemodel, check that we have a non-null aabb if the .rivescene didn't have one.